src/components/BillCard/BillCard.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+	import { ArrowRight } from 'carbon-icons-svelte';
+
+	import BillStatusTag from '$components/BillStatusTag/BillStatusTag.svelte';
+	import type { BillStatus } from '$models/bill';
+
+	interface ProposedBy {
+		avatar: string;
+		name: string;
+		description: string;
+	}
+
+	// see: https://github.com/wevisdemo/parliament-watch/issues/26#issue-1937096920
+	export let orientation: 'landscape' | 'portrait' = 'landscape';
+	export let nickname: string;
+	export let title: string;
+	export let proposedBy: ProposedBy;
+	export let proposedOn: Date;
+	export let status: BillStatus;
+	export let currentState: string | undefined;
+	export let daySinceProposed: number;
+	export let billUrl: string;
```

Comment:
We use `orientation === 'landscape'` condition so many times. I suggest adding another reactive variable here

```svelte
$: isLandscape = orientation === 'landscape'
```

So that  we can replace `orientation === 'landscape'` with `isLandscape` in the whole file